### DISCUSSION TITLE
Jump to next occurence of LSP hovered word?

### DISCUSSION THREAD

**Discussion by druskus20** [LazyVim/LazyVim#5136] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/d274928e-9141-448c-8bab-2bda875acbed)

I am trying to find a way to map `n` and `N` to "go to the next occurrence of the hovered word". Does anyone know any way to do this?

**Comment by gengdz** [comment, marked as answer]
try ]]

**Reply by druskus20** [reply]
Oh wow - yeah that's simple. I never use that :) 
Thanks!
